addons/product_extended/models/product.py

```diff
@@ -37,9 +37,7 @@ def _calc_price(self, bom):
         price = 0.0
         result, result2 = bom.explode(self, 1)
         for sbom, sbom_data in result2:
-            if not sbom.attribute_value_ids:
-                # No attribute_value_ids means the bom line is not variant specific
-                price += sbom.product_id.uom_id._compute_price(sbom.product_id.standard_price, sbom.product_uom_id) * sbom_data['qty']
+            price += sbom.product_id.uom_id._compute_price(sbom.product_id.standard_price, sbom.product_uom_id) * sbom_data['qty']
```

Comment:
Taking nothing might indeed be smarter than taking it all, but it changes behaviour and the value is underestimated.  (but I could agree with this)